libs/composer/only-in-dev.php

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types=1);
+/* Copyright (c) 2020 ILIAS open source, Extended GPL, see docs/LICENSE */
```

Comment:
@lscharmer Can you please replace the PHP header (see: https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/CI/Rector/ChangeLicenseHeader.php#L15)?